client/src/components/payment/payment-modal.vue

```diff
@@ -4,7 +4,6 @@
 
       <v-card-title class="d-flex justify-space-between">
         <div>Confirm Name Request</div>
-        <countdown-timer :timerName="timerName" colorString="#003366" bgColorString="#efefef" style="float: right"/>
```

Comment:
I suggest you add the close "X" here instead (as on the other similar modals). Check with Scott if needed.